### DISCUSSION TITLE
Support unix sockets

### DISCUSSION THREAD

**Discussion by mike-vermillio** [ory/hydra#2897] [Q&A] [answered]
I can't quite tell if hydra supports unix sockets for database connections? I don't see a lot of documentation and when digging into the shared library that parses the postgres connection url I'm not seeing anything.

**Comment by vinckr** [comment, marked as answer]
Hey mike 👋 ,
it should be possible. 
Check out this previous discussion on our old forum for some pointers: 
https://community.ory.sh/t/how-to-connect-to-postgresql-via-unix-socket/2211/3
(It is about Ory Kratos, but should work the same for Hydra)

**Reply by mike-vermillio** [reply]
Ah ya that's exactly what I was looking for, thanks!

**Reply by vinckr** [reply]
Great 🙌 
From the top of your head, can you think of a location in the documentation where we could add this?

**Reply by mike-vermillio** [reply]
This seems like a reasonable place: https://www.ory.sh/docs/ecosystem/deployment/ 
It's not directly under Hydra but links from the Hydra database configuration page. 

I ended up trying MySQL and was able to get unix sockets working there too:
`DNS=mysql://${username}:${password}@unix(${unix_path})/${database}?parseTime=true`
